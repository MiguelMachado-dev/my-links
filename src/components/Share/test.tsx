import { describe, it, expect } from 'vitest'
import { render, screen } from '@testing-library/react'
import { Share } from '.'

describe('<Share />', () => {
  it('should render <Share /> correctly', () => {
    render(<Share />)

    expect(
      screen.getByRole('heading', {
        name: /compartilhe:/i,
      })
    ).toBeInTheDocument()

    expect(
      screen.getByRole('heading', {
        name: /compartilhe:/i,
      })
    ).toMatchSnapshot()
  })
})
