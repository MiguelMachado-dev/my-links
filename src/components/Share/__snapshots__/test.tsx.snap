// Vitest Snapshot v1

exports[`<Share /> > should render <Share /> correctly 1`] = `
.c0 {
  font-size: 1.4rem;
  line-height: 1.5;
  margin-bottom: 0.6rem;
  text-align: center;
}

<h3
  class="c0"
>
  Compartilhe:
</h3>
`;
